group-coordinator/src/test/java/org/apache/kafka/coordinator/group/consumer/TargetAssignmentBuilderTest.java

```diff
@@ -203,7 +209,10 @@ public TargetAssignmentBuilder.TargetAssignmentResult build() {
 
             // Verify that the assignor was called once with the expected
             // assignment spec.
-            verify(assignor, times(1)).assign(assignmentSpec);
+            verify(assignor, times(1))
+                .assign(
+                    assignmentSpec, assignmentTopicMetadata
+                );
```

Comment:
nit: This code format is weird. If you really want to break the line, I would format it as follow:

```
verify(assignor, times(1)).assign(
    assignmentSpec,
    assignmentTopicMetadata
);
```

or as follow (like you did at L189)

```
verify(assignor, times(1))
    .assign(assignmentSpec, assignmentTopicMetadata);
```